verticapy/ml_pipeline/_train.py

```diff
@@ -0,0 +1,145 @@
+#!/usr/bin/env python3
+"""
+Copyright  (c)  2018-2024 Open Text  or  one  of its
+affiliates.  Licensed  under  the   Apache  License,
+Version 2.0 (the  "License"); You  may  not use this
+file except in compliance with the License.
+
+You may obtain a copy of the License at:
+    http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless  required  by applicable  law or  agreed to in
+    writing, software  distributed  under the  License is
+    distributed on an  "AS IS" BASIS,  WITHOUT WARRANTIES
+    OR CONDITIONS OF ANY KIND, either express or implied.
+    See the  License for the specific  language governing
+    permissions and limitations under the License.
+"""
+"""
+This script runs the Vertica Machine Learning Pipeline Training.
+"""
+from typing import Tuple
+
+from verticapy.core.vdataframe.base import vDataFrame
+from verticapy._typing import SQLColumns
+from verticapy.machine_learning.vertica.base import VerticaModel
+
+from verticapy.machine_learning.vertica import *
```

Comment:
avoid using "*" for the import and all the models are in verticapy.machine_learning.vertica, no need to go to a more nested structure.